tests/Aspire.Hosting.Oracle.Tests/OracleContainerFixture.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Aspire.Hosting.Testing;
+using Aspire.Hosting.Utils;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Polly;
+using Xunit;
+
+namespace Aspire.Hosting.Oracle.Tests;
+public sealed class OracleContainerFixture : IAsyncLifetime
```

Comment:
What's the point of this fixture?

Reply:
Start a docker container such that the image is available before any actual test starts. Otherwise each pipeline needs to account for the fact that the image could take 10mins to download start. Also can't be in a constructor since it's all async.

Reply:
It would be good to add this reasoning to a comment. As it stands, it looks like this isn't doing anything other than taking more time.